apps/web/src/components/NavBar/ChainSelector.tsx

```diff
@@ -146,7 +146,6 @@ export const ChainSelector = ({ leftAlign }: { leftAlign?: boolean }) => {
     width: 20,
     color: theme.neutral2,
   }
-
```

Comment:
maybe we should avoid adding these which could potentially add rebasing conflicts in the future (if we ever rebase)

Reply:
removed here: https://github.com/hemilabs/interface/pull/1/commits/447c41cd5e18dbf107c45eea506c98090550a543